### DISCUSSION TITLE
build process returns an error which is unclear, and wasn't happened before.

### DISCUSSION THREAD

**Discussion by MinhoChoi-a** [codemagic-ci-cd/codemagic-docs#1974] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. There was error during ios app build process.
2. so I tried same code(which was success on build and publish few days ago) to build.
3. but it also returns same error message.

### Expected results

Build process should be okay, as it was working with same code(I tried a specific git commit which went through for all process)

### Actual results

Xcode archive done.                                         261.9s
Failed to build iOS app
Error (Xcode): ../.pub-cache/hosted/pub.dev/sliver_tools-0.2.10/lib/src/sliver_stack.dart:282:55: Error: A value of type 'RenderObject?' can't be assigned to a variable of type 'AbstractNode?'.

Encountered error while archiving for device.

### Build id (optional)

64de2c0c234c6e0c0b23306b

**Comment by icarusdust** [comment]
Hi @MinhoChoi-a, looks like it is referring to your code where RenderObject is defined and I am assuming that you are facing this issue when building locally as well. Check out [this thread](https://github.com/flutter/flutter/issues/82254).

**Reply by rigaldamez** [reply]
Hi @MinhoChoi-a  - Did you manage to fix this?

I am encountering exact same problem.

**Comment by icarusdust** [comment]
I am closing this thread due to inactivity. If you tried the solution mentioned in the thread shared above, and it did not help, you can reopen this thread.
